app/src/lib/git/for-each-ref.ts

```diff
@@ -153,45 +150,41 @@ export async function getBranchesDifferingFromUpstream(
   // - For local branches with upstream: name, ref, SHA and the upstream.
   // - For remote branches we only need the sha (and the ref as key).
   for (const line of lines) {
-    const [ref, name, sha, upstream, symref, head] = line.split('\0')
+    const [ref, sha, upstream, symref, head] = line.split('\0')
 
     if (symref.length > 0 || head === '*') {
       // Exclude symbolic refs and the current branch
       continue
     }
 
-    if (ref.startsWith('refs/head')) {
+    if (ref.startsWith('refs/heads')) {
       if (upstream.length === 0) {
         // Exclude local branches without upstream
         continue
       }
 
-      localBranches.push({ name, ref, sha, upstream })
+      localBranches.push({ ref, sha, upstream })
     } else {
       remoteBranchShas.set(ref, sha)
     }
   }
 
-  const eligibleBranchNames = new Set<String>()
+  const eligibleBranches: ITrackingBranch[] = []
```

Comment:
These are semantically equivalent but we typically use the 👇  syntax nowadays

```suggestion
  const eligibleBranches = new Array<ITrackingBranch>()
```